packages/flutter/lib/src/foundation/platform.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The platform that user interaction should adapt to target.
+enum TargetPlatform {
```

Comment:
We can always introduce another "version" enum if we want to get more specific in our targetting.